src/prefect/server/database/migrations/versions/postgresql/2023_04_05_122716_15f5083c16bd_migrate_artifact_data.py

```diff
@@ -0,0 +1,67 @@
+"""Migrate artifact data to artifact_collection table
+
+Revision ID: 15f5083c16bd
+Revises: 6a1eb3d442e4
+Create Date: 2023-04-05 12:27:16.676260
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "15f5083c16bd"
+down_revision = "6a1eb3d442e4"
```

Comment:
I think this might need a merge from main + change to a different down migration